taskcluster/ci/ui-test/kind.yml

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+    - build
+
+job-defaults:
+    attributes:
+        build-type: debug
+        code-review: true
+        retrigger: true
+    # dependencies:
+    #     build-debug: build-focus-debug
+    #     build-android-test: build-android-test-debug
+    fetches:
+        toolchain:
+            - android-sdk-linux
+            - android-gradle-dependencies
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 7200
+        env:
+            GOOGLE_APPLICATION_CREDENTIALS: '.firebase_token.json'
+            GOOGLE_PROJECT: 'moz-fx-mobile-firebase-testlab'
+        artifacts:
+            - name: public
+              path: /builds/worker/artifacts
+              type: directory
+    run-on-tasks-for: [github-pull-request, github-push]
+    treeherder:
+        kind: test
+        platform: 'ui-test/opt'
+        tier: 2
+    run:
+        use-caches: false
+        using: gradlew
+        dummy-secrets:
+            - content: "faketoken"
+              path: .adjust_token
+        secrets:
+            - name: project/mobile/focus-android/firebase
+              key: firebaseToken
+              path: .firebase_token.json
+              json: true
+        # TODO retrieving these from two separate tasks means they're signed differently; need to create a signing
```

Comment:
I assume this TODO is still relevant or was this resolved?

Reply:
Its a TODO for once this gets merged and we merge ahal's signing kind. So we'll switch over to before, where we use the focusDebug build that ran in a separate task, and then run the androidTest in a separate task to speed things up, but the dep-signing will resolve the issue with the apks being signed differently.